tests/laravel-test.sh

```diff
@@ -18,8 +18,11 @@ sed -e 's|@return \\Illuminate\\Http\\Response$|@return \\Symfony\\Component\\Ht
 
 echo "Fix https://github.com/nunomaduro/larastan/pull/378#issuecomment-565706907"
 sed -e 's/string/string|void/' -i app/Http/Middleware/Authenticate.php
-sed '0,/}/s/}/}\nreturn;/' -i app/Http/Middleware/Authenticate.php
+sed -e '0,/}/s/}/}\nreturn;/' -i app/Http/Middleware/Authenticate.php
 
+echo "Fix commented namespae property in RouteServiceProvider"
```

Comment:
typo: namespace

Reply:
Thank you.

Reply:
Fixed in master https://github.com/nunomaduro/larastan/pull/1107/files